meal-planning/src/components/recipe-library/RecipesLibrary.js

```diff
@@ -69,7 +44,13 @@ const RecipesLibrary = () => {
       </div>
 
       <div className={styles.recipeSearchBar}>
-        <TextField fullWidth="true" id="outlined-basic"  variant="outlined" placeholder = "Search by recipe or tag ..." onChange = {e => setSearch(e.target.value)}/>
+        <TextField
+          fullWidth="true"
```

Comment:
needs to just be `fullwidth `or `fullwidth={true}`. I'd say do the former since that's what the others are.